### ISSUE TITLE
slcli vlan detail versus subnet detail

### ISSUE THREAD

**Issue by mweisbac** [softlayer/softlayer-python#833]
The slcli output for "vlan detail" and "subnet detail" differs. While "vlan detail" provides "hostname, domain, public_ip, private_ip" for vs and hardware, the "subnet detail" provides "Hostname, Domain, IP" (where IP is the public one).

### Expected Behavior
"subnet detail" shows "hostname, domain, public_ip, private_ip" too, 
so it's much easier to get an overview about a subnet / IP range and which IPs are in use, for both subnets on private and public VLANs

### Actual Behavior
"subnet detail" shows "Hostname, Domain, IP"

### Environment Information
Operating System:
softlayer-python version (`slcli --version`): 5.2.6

**Comment by mweisbac** [comment]
```
--- SoftLayer/CLI/subnet/detail.py.orig	2017-05-19 16:45:32.093380212 +0200
+++ SoftLayer/CLI/subnet/detail.py	2017-05-31 21:11:26.863536707 +0200
@@ -47,26 +47,28 @@
 
     if not no_vs:
         if subnet['virtualGuests']:
-            vs_table = formatting.Table(['Hostname', 'Domain', 'IP'])
+            vs_table = formatting.Table(['hostname', 'domain', 'public_ip', 'private_ip'])
             vs_table.align['Hostname'] = 'r'
             vs_table.align['IP'] = 'l'
             for vsi in subnet['virtualGuests']:
                 vs_table.add_row([vsi['hostname'],
                                   vsi['domain'],
-                                  vsi.get('primaryIpAddress')])
+                                  vsi.get('primaryIpAddress'),
+                                  vsi.get('primaryBackendIpAddress')])
             table.add_row(['vs', vs_table])
         else:
             table.add_row(['vs', 'none'])
 
     if not no_hardware:
         if subnet['hardware']:
-            hw_table = formatting.Table(['Hostname', 'Domain', 'IP'])
+            hw_table = formatting.Table(['hostname', 'domain', 'public_ip', 'private_ip'])
             hw_table.align['Hostname'] = 'r'
             hw_table.align['IP'] = 'l'
             for hardware in subnet['hardware']:
                 hw_table.add_row([hardware['hostname'],
                                   hardware['domain'],
-                                  hardware.get('primaryIpAddress')])
+                                  hardware.get('primaryIpAddress'),
+                                  hardware.get('primaryBackendIpAddress')])
             table.add_row(['hardware', hw_table])
         else:
             table.add_row(['hardware', 'none'])

**Comment by sergiocarlosmorales** [comment]
Ack, thanks!
I went ahead and implemented your change set, along with other changes to fix inconsistencies with VLAN detail output (capitalization & alignment) in pull request https://github.com/softlayer/softlayer-python/pull/851

Seems these 2 share a lot of common data points, so I'd like to unify the output at some point, will leave that for another time.

**Comment by sergiocarlosmorales** [comment]
@mweisbac change set has been merged & released as 5.2.11
Thanks!
